### DISCUSSION TITLE
Detected by X

### DISCUSSION THREAD

**Discussion by alicesin615** [seleniumbase/SeleniumBase#3095] [Q&A] [answered]
Was testing the login flow on X and unfortunately they seemed to have detected me as a bot as they're now displaying the "Suspicous activity detected...". Does seleniumbase currently support a randomized humanized behaviour? I suspect the detection might be because of how the actions triggered were too sequential.

**Comment by mdmintz** [comment, marked as answer]
You may be able to avoid issues by using their API directly: https://developer.x.com/en/docs/x-api.
For randomization, you can try adding your own randomness. (Python has libraries for that.)
But if Twitter detects too much traffic, then the issue is the same as https://github.com/seleniumbase/SeleniumBase/issues/2291.
You can change your proxy settings with the `proxy` arg to get around that.
